packages/coinstac-pipeline/src/pipeline-manager.js

```diff
@@ -277,8 +320,6 @@ module.exports = {
           throw new Error(`Unable to create pipeline directories: ${err}`);
         })
         .then(() => {
-          activePipelines[runId].state = 'running';
```

Comment:
pipeline-manager pipeline state is actually not used anywhere, the UI state comes from the controller. Even so I thought it was useful to setup some state for the future, unfortunately this didnt work with everywhere with the `pre-pipeline` state solution /shrug